l10n_de_tax_statement/models/l10n_de_tax_statement_2019.py

```diff
@@ -0,0 +1,346 @@
+                  'Steuerbeträge gem. § 14c UstG) sowie Steuerbetr. d. n. '
+                  '§ 6a Abs. 4 S. 2, § 17 Abs. 1 S. 6, § 25 b Abs. 2 UStG '
+                  'o. v. e. Auslagerer o. Lagerh. n. § 13a Abs. 1 Nr. 6 '
+                  'UStG geschuldet werden (69)'),
+    },
+    '64': {
+        'code': '64', 'tax': 0.0,
+        'name': _('Umsatzsteuer-Vorauszahlung'),
+    },
+    '65': {
+        'code': '65', 'tax': 0.0,
+        'name': _('Abzug der festges. Sondervorauszahl. f. '
+                  'Dauerfristverlängerung, nur auszuf. i. d. letzten '
+                  'Voranmeldung d. Besteuerungszeitr., i.d.R. Dez. (39)'),
+    },
+
+}
+
+
+def _finalize_lines_2019(lines):
+    _26b = lines['26']['tax']
+    _27b = lines['27']['tax']
+    _28b = lines['28']['tax']
+    _64b = lines['64']['tax']
+    _65b = lines['65']['tax']
+    # calculate lines 48 - 50
+    lines['48']['tax'] = lines['48']['tax'] * 1
+    _48b = lines['48']['tax']
+    lines['49']['tax'] = lines['49']['tax'] * 1
+    _49b = lines['49']['tax']
+    lines['50']['tax'] = lines['50']['tax'] * 1
+    _50b = lines['50']['tax']
+    # calculate lines 26, 27, 28, 33, 34
+    lines['26']['tax'] = lines['26']['base'] * 0.19
+    _26b = lines['26']['tax']
+    lines['27']['tax'] = lines['27']['base'] * 0.07
+    _27b = lines['27']['tax']
+    lines['30']['tax'] = lines['30']['base'] * 0.19
+    _30b = lines['30']['tax']
+    lines['33']['tax'] = lines['33']['base'] * 0.19
+    _33b = lines['33']['tax']
+    lines['34']['tax'] = lines['34']['base'] * 0.07
+    _34b = lines['34']['tax']
+    # calculate reverse of lines 32 - line 36 base
+    lines['32']['base'] = lines['32']['base'] * -1
+    lines['33']['base'] = lines['33']['base'] * -1
+    lines['34']['base'] = lines['34']['base'] * -1
+    lines['35']['base'] = lines['35']['base'] * -1
+    lines['36']['base'] = lines['36']['base'] * -1
+    # calculate reverse of lines 32 - line 36 tax
+    lines['33']['tax'] = lines['33']['tax'] * -1
+    _33b = lines['33']['tax']
+    lines['34']['tax'] = lines['34']['tax'] * -1
+    _34b = lines['34']['tax']
+    lines['35']['tax'] = lines['35']['tax'] * -1
+    _35b = lines['35']['tax']
+    lines['36']['tax'] = lines['36']['tax'] * -1
+    _36b = lines['36']['tax']
+    # calculate reverse of lines 48 - line 50 base
+    lines['48']['base'] = lines['48']['base'] * -1
+    lines['49']['base'] = lines['49']['base'] * -1
+    lines['50']['base'] = lines['50']['base'] * -1
+    # calculate reverse of line 53 - line 59
+    lines['53']['tax'] = lines['53']['tax'] * -1
+    _53b = lines['53']['tax']
+    lines['54']['tax'] = lines['54']['tax'] * -1
+    _54b = lines['54']['tax']
+    lines['55']['tax'] = lines['55']['tax'] * -1
+    _55b = lines['55']['tax']
+    lines['56']['tax'] = lines['56']['tax'] * -1
+    _56b = lines['56']['tax']
+    lines['57']['tax'] = lines['57']['tax'] * -1
+    _57b = lines['59']['tax']
```

Comment:
Should _57b be renamed to _59b?

Reply:
No that was not my intention. I have corrected those values.